src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ContextInfos/VersionInfoContextInfo.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Sort / remove usings

Reply:
Sort/remove should not cause additional using statements. 
If there's no reason for that, revert this file.